EIPS/eip-eth_simulateTransaction.md

```diff
@@ -0,0 +1,109 @@
+---
+eip: <to be assigned>
+title: eth_simulateTransaction - transaction dry runs
+author: Peter Jihoon Kim (@petejkim)
+discussions-to: <tbd>
+status: Draft
+type: Standards Track
+category (*only required for Standard Track): Interface
+created: 2020-03-16
+---
+
+## Simple Summary
+
+Wallet/DApp browser software should provide as much information as possible when presenting a transaction to the user to sign, as there may be unforeseen consequences including a potential loss of funds. A new JSONRPC method `eth_simulateTransaction` enables wallets to perform a dry run of the transaction so that it can provide more information about what might happen such as token movements.
```

Comment:
Recommend moving most of this into another section such as the Abstract and/or Motivation.  Simple summary should be about a sentence, and not a long one.  When someone asks, "what is EIP 2560" this would be the one-line answer, while the Abstract would describe the specification at a high level in human readable terms, and the motivation section describes **why** this EIP is desirable.